pyresample/bucket/__init__.py

```diff
@@ -126,8 +126,8 @@ def _get_indices(self):
         # Calculate array indices. Orient so that 0-meridian is pointing down.
         adef = self.target_area
         x_res, y_res = adef.resolution
-        x_idxs = da.floor((proj_x - adef.area_extent[0]) / x_res).astype(np.int)
-        y_idxs = da.floor((adef.area_extent[3] - proj_y) / y_res).astype(np.int)
+        x_idxs = da.floor((proj_x - adef.area_extent[0]) / x_res).astype(int)
+        y_idxs = da.floor((adef.area_extent[3] - proj_y) / y_res).astype(int)
```

Comment:
Do you want these to be `np.int64` too?

Reply:
Yeah. Thanks, missed them in the previous change :man_facepalming: